### PR TITLE
DOCSUP-14792-document-bitShift-functions

### PR DESCRIPTION
**Pull request by lehasm**
Changelog category:

- Documentation for #28325

### CODE REVIEW

**Comment by lehasm** [comment]
Hi.
The doc stated that _Bit functions work for any pair of types from UInt8, ... Float32, or Float64._
https://clickhouse.com/docs/en/sql-reference/functions/bit-functions

Tried a query with floats but it raised an exception.
`SELECT bitAnd(toFloat32(99), toFloat32(15));`

What does the doc mean? Is it correct?

File: docs/ru/sql-reference/functions/bit-functions.md [hunk 1]
```diff
@@ -19,8 +19,101 @@ toc_title: "Битовые функции"
...
+┌─a───┬─hex(toFixedString('abc', 3))─┬─a_shifted─┬─hex(bitShiftLeft(toFixedString('abc', 3), 4))─┐
+│ abc │ 616263                       │ &0        │ 162630                                        │
+└─────┴──────────────────────────────┴───────────┴───────────────────────────────────────────────┘
+```
+
 ## bitShiftRight(a, b) {#bitshiftrighta-b}
 
+Сдвигает вправо на заданное количество битов значение, представленное в бинарном виде.
+
+Если передан аргумент типа `FixedString` или `String`, то он рассматривается, как одно многобайтовое значение.
```
Comment (gyuton) [inline docs/ru/sql-reference/functions/bit-functions.md:73]:
```suggestion
Если передан аргумент типа `FixedString` или `String`, то он рассматривается, как одно многобайтовое значение.```

File: docs/ru/sql-reference/functions/bit-functions.md [hunk 2]
```diff
@@ -19,8 +19,101 @@ toc_title: "Битовые функции"
...
+│ abc │ 616263                       │ &0        │ 162630                                        │
+└─────┴──────────────────────────────┴───────────┴───────────────────────────────────────────────┘
+```
+
 ## bitShiftRight(a, b) {#bitshiftrighta-b}
 
+Сдвигает вправо на заданное количество битов значение, представленное в бинарном виде.
+
+Если передан аргумент типа `FixedString` или `String`, то он рассматривается, как одно многобайтовое значение. 
+Длина значения типа `String` уменьшается по мере сдвига.
```
Comment (gyuton) [inline docs/ru/sql-reference/functions/bit-functions.md:74]:
```suggestion
```

File: docs/ru/sql-reference/functions/bit-functions.md [hunk 3]
```diff
@@ -19,8 +19,101 @@ toc_title: "Битовые функции"
 
 ## bitShiftLeft(a, b) {#bitshiftlefta-b}
 
+Сдвигает влево на заданное количество битов значение, представленное в бинарном виде.
```
Comment (gyuton) [inline docs/ru/sql-reference/functions/bit-functions.md:22]:
```suggestion
Сдвигает влево на заданное количество битов бинарное представление значения.
```

File: docs/ru/sql-reference/functions/bit-functions.md [hunk 4]
```diff
@@ -19,8 +19,101 @@ toc_title: "Битовые функции"
...
+│ abc │ 616263     │ &0        │ 06162630                    │
+└─────┴────────────┴───────────┴─────────────────────────────┘
+┌─a───┬─hex(toFixedString('abc', 3))─┬─a_shifted─┬─hex(bitShiftLeft(toFixedString('abc', 3), 4))─┐
+│ abc │ 616263                       │ &0        │ 162630                                        │
+└─────┴──────────────────────────────┴───────────┴───────────────────────────────────────────────┘
+```
+
 ## bitShiftRight(a, b) {#bitshiftrighta-b}
 
+Сдвигает вправо на заданное количество битов значение, представленное в бинарном виде.
```
Comment (gyuton) [inline docs/ru/sql-reference/functions/bit-functions.md:71]:
```suggestion
Сдвигает вправо на заданное количество битов бинарное представление значения.
```

**Comment by gyuton** [comment]
@CurtizJ Hi! Can you review this, please?

**Review by CurtizJ — APPROVED** [review]

**Pull request merged**